lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -79,22 +82,36 @@ def heredoc?(node)
           args.loc.expression.source[0, 2] == SHOVEL
         end
 
+        def literal?(node)
```

Comment:
I believe we have this method is a few cops now, so it should probably be extracted in some helper module (either existing or a new one).